Controllers/DayOffRequestController.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using stunning_robot_HR.Data;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Controllers
+{
+    public class DayOffRequestController : Controller
+    {
+        private readonly stunning_robot_HRContext _context;
+
+        public DayOffRequestController(stunning_robot_HRContext context)
+        {
+            _context = context;
+        }
+
+        // GET: DayOffRequest
+        public async Task<IActionResult> Index()
+        {
+            return View(await _context.DayOffRequests.ToListAsync());
+        }
+
+        // GET: DayOffRequest/Details/5
+        public async Task<IActionResult> Details(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var dayOffRequest = await _context.DayOffRequests
```

Comment:
One of those spots where you should strive for single responsibility principle. I would traditionally separate my queries into a diffierent class. A repository for the repository pattern. A Query or Command in CQS, ect ect.